components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -46,6 +45,7 @@
 import org.wso2.carbon.identity.oauth2.token.OAuthTokenReqMessageContext;
 import org.wso2.carbon.identity.oauth2.token.OauthTokenIssuer;
 import org.wso2.carbon.identity.oauth2.token.bindings.TokenBinding;
+import org.wso2.carbon.identity.oauth2.util.JdbcUtils;
```

Comment:
Most of the duplicate JdbcUtils have been deprecated with https://github.com/wso2/carbon-identity-framework/pull/3923. Need to check whether the OAuth one was kept intentionally.

Reply:
Actually there were to `JdbcUtils`, 
1. `org.wso2.carbon.identity.application.mgt.util.JdbcUtils` and 
2. `org.wso2.carbon.identity.oauth2.util.JdbcUtils`
I saw method(s) used by `1.` in this class was deprecated (therefore replaced it) and besides why not use the `2.` from the same component instead of referring another component.

Reply:
Actually there were 2 `JdbcUtils` classes, 
1. `org.wso2.carbon.identity.application.mgt.util.JdbcUtils` and 
2. `org.wso2.carbon.identity.oauth2.util.JdbcUtils`
Saw method(s) used by `1.` in this class was deprecated (therefore replaced it) and besides why not use the `2.` from the same component instead of referring another component.